Sources/ProjectDescription/RunActionOptions.swift

```diff
@@ -87,76 +87,6 @@ public struct RunActionOptions: Equatable, Codable {
     }
 }
 
-extension RunActionOptions {
-    /// Simulated location represents a GPS location that is used when running an app on the simulator.
-    public struct SimulatedLocation: Codable, Equatable {
```

Comment:
This is technically a breaking change. What do you think of defining a deprecated typealias to ensure we don't break anyone's manifests?
```swift
extension RunActionOptions {
  @available(*, unavailable, message: "Use ProjectDescription.SimulatedLocation directly.")
  typealias SimulatedLocation = ProjectDescription.SimulatedLocation
}
```

Reply:
I didn't realize that was a breaking change, thanks for checking it out. However, what do you think about `deprecated` instead of `unavailable`? I think compile error might imply that the feature has changed.

Reply:
Yeah, sure!